src/DefaultExceptionHandler.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+
+namespace Amp\Http\Server;
+
+use Amp\Http\HttpStatus;
+use Psr\Log\LoggerInterface as PsrLogger;
+
+/**
+ * Simple exception handler that writes a message to the logger and returns an error page generated by the provided
+ * {@see ErrorHandler}.
+ */
+final class DefaultExceptionHandler implements ExceptionHandler
+{
+    public function __construct(
+        private readonly ErrorHandler $errorHandler,
+        private readonly PsrLogger $logger,
+    ) {
+    }
+
+    public function handleException(\Throwable $exception, Request $request): Response
```

Comment:
I'd put the `Request` as first parameter, I think.

Reply:
Swapped.